region/norfair/east/Acid Snakes Tunnel.json

```diff
@@ -115,13 +117,126 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 2],
+      "name": "Come in Shinecharging, Leave With Temporary Blue",
+      "entranceCondition": {
+        "comeInShinecharging": {
```

Comment:
You can spacejump through [1, 2] to leave speedballing and that would be a comeInGettingBlueSpeed.
These strats make you do a Temporary Blue chain so there is no real difference (with comeInShinecharing)